src/validators/debug/index.js

```diff
@@ -1,6 +1,12 @@
-import {noop} from 'lodash'
+import {isBoolean} from 'lodash'
 export default {
   key: 'debug',
-  validate: noop,
+  validate: validateDebug,
+}
+
+function validateDebug(val) {
+  if (!isBoolean(val)) {
+    return 'no boolean value provided'
```

Comment:
Could you change this to: `'must be a boolean value'`?

I would recommend that you do:

1. Make your changes
2. Run `git add src/validators/debug/index.js`
3. Run `git commit --amend`
4. Run `git push -f` (`-f` is to force push to your branch because `--amend` will change your git history).
5. Let me know you've updated the PR (Because github wont let me know automatically).

Thanks!